src/test/compile-fail/borrowck-closures-mut-of-imm.rs

```diff
@@ -24,6 +24,7 @@ fn a(x: &int) {
     //~^ ERROR cannot borrow
     let c2 = || set(&mut *x);
     //~^ ERROR cannot borrow
```

Comment:
I purposefully keep the expected messages short. I've spent far too many hours of my life tweaking the error messages in compile-fail tests.